.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,78 @@
+name: "🐞 Bug report"
+description: "Report a bug if something isn't working as expected in the Elementor"
+title: "🐞 Bug report: "
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## BEFORE POSTING YOUR ISSUE
+
+        - Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!
+
+        - If you have questions or need general support,  Please use:  https://wordpress.org/support/plugin/elementor
+
+        - For help and support from the Elementor community, see: https://www.facebook.com/groups/Elementors/
+
+        - To read more about Elementor, check out our documentation: https://docs.elementor.com
+
+        - Developers docs are located at https://developers.elementor.com/
+
+        ### Guidelines
+
+        - Search this repository (top of the page) for the issue, and make sure it has not been fixed or reported before.
```

Comment:
Maybe?

```suggestion
        - Search this repository (top of the page) for the issue, and make sure it has not been fixed or reported already.
```

Reply:
I think `before` is more appropriate here.